src/webui/webapplication.cpp

```diff
@@ -451,6 +472,11 @@ Http::Response WebApplication::processRequest(const Http::Request &request, cons
     if (!m_contentSecurityPolicy.isEmpty())
         header(QLatin1String(Http::HEADER_CONTENT_SECURITY_POLICY), m_contentSecurityPolicy);
 
+    if (m_useCustomHTTPHeaders) {
+        for (const CustomHTTPHeader &i : asConst(m_customHTTPHeaders))
```

Comment:
Why "i"? Why it's not "header"?

Reply:
>Why it's not "header"?

It will cause a name clash with header() function. I renamed it to something else.